flowey/flowey_lib_hvlite/src/artifact_openhcl_igvm_from_recipe_extras.rs

```diff
@@ -68,7 +68,7 @@ pub mod publish {
                             let OpenvmmHclOutput { bin, dbg } = openvmm_hcl_bin;
                             files.push((format!("{folder_name}/openhcl").into(), bin));
```

Comment:
lets also change the exe name as well. it should be `openhcl_vmm`

Reply:
```suggestion
                            files.push((format!("{folder_name}/openvmm_hcl").into(), bin));
```